lib/web_console/middleware.rb

```diff
@@ -2,13 +2,6 @@
 
 module WebConsole
   class Middleware
-    TEMPLATES_PATH = File.expand_path('../templates', __FILE__)
```

Comment:
The constant seems unused. Moreover, Template class has a similar class attribute, so it might make sense to combine them into one? We could access the attribute from Middleware class if we needed.

But I feel it goes off-topic, so I will reset the line and create another pull request later if needed :-)